src/verifySignature.ts

```diff
@@ -0,0 +1,60 @@
+import { colors, flags } from './deps.ts'
+import { exit, revokeNet } from './utils.ts'
+import { deserializeKey, keyId, verifySignature as cryptoVerifySignature } from './lib/crypto.ts'
+import { importJsonFile } from './utils.ts'
+
+export const verifySignature = async (args: string[], internal = false) => {
+  await revokeNet()
+  const parsedArgs = flags.parse(args)
+  const [manifestFile] = parsedArgs._
+  const keyFile = parsedArgs.k
+  const [externalKeyDescriptor, manifest] = await Promise.all([
+    keyFile ? importJsonFile(keyFile) : null,
+    importJsonFile(manifestFile)
+  ])
+  if (keyFile && !externalKeyDescriptor.pubkey) {
+    exit('Public key missing from key file', internal)
+  }
+  if (!manifest.head?.manifestVersion) {
+    exit('Invalid manifest: missing head or manifest version', internal)
+  }
+  if (!manifest.body) {
+    exit('Invalid manifest: missing body', internal)
+  }
+  if (!manifest.signature) {
+    exit('Invalid manifest: missing signature', internal)
+  }
+  if (!manifest.signature.keyId) {
+    exit('Invalid manifest: missing signature key ID', internal)
+  }
+  if (!manifest.signature.value) {
+    exit('Invalid manifest: missing signature value', internal)
+  }
```

Comment:
We also need to load both contract files (slim and bundle) and verify the hash matches the hash that's in the manifest.